examples/guestbook/README.md

```diff
@@ -323,19 +330,54 @@ if (isset($_GET['cmd']) === true) {
 } ?>
 ```
 
-To play with the service itself, find the name of a frontend, grab the external IP of that host from the [Google Cloud Console][cloud-console] or the `gcloud` tool, and visit `http://<host-ip>:8000`.
+### Step Six: Create the guestbook service.
+
+Just like the others, you want a service to group your frontend pods.
+
+The service is described in the file `examples/guestbook/frontend-service.json`:
+
+```js
+{
+  "id": "frontend",
+  "kind": "Service",
+  "apiVersion": "v1beta1",
+  "port": 80,
+  "containerPort": "http-server",
+  "selector": {
+    "name": "frontend"
+  },
+  "labels": {
+    "name": "frontend"
+  },
+  "createExternalLoadBalancer": true
+}
+```
+
+```shell
+$ cluster/kubectl.sh create -f examples/guestbook/guestbook-service.json
+
+$ cluster/kubectl.sh get services
+NAME                    LABELS                                    SELECTOR                     IP                  PORT
+frontend                name=frontend                             name=frontend                10.0.93.211         80
+redis-master            name=redis-master                         name=redis-master            10.0.246.242        6379
+redisslave              name=redis-slave                          name=redis-slave             10.0.72.62          6379
+```
+
+You may need to open the firewall for port 80 using the [console][cloud-console] or the `gcloud` tool. The following command will allow traffic from any source to instances tagged `kubernetes-minion`:
 
 ```shell
-$ gcloud compute instances list
+$ gcloud compute firewall-rules create --allow=tcp:80 --target-tags=kubernetes-minion kubernetes-minion-80
```

Comment:
Fix the service with port 8000, instead of requiring 80?  We'd better not use priviledge port.